scripts/contentFixer.lua

```diff
@@ -106,44 +110,47 @@ function contentFixer.AdjustSharedCorprusState(pid)
     end
 end
 
+---@param journal table
+---@return boolean madeAdjustment Has cell lost/gained corprus
 function contentFixer.AdjustWorldCorprusVariables(journal)
-
-    local madeAdjustment = false
-
     for _, journalItem in ipairs(journal) do
-
         if journalItem.quest == "a2_3_corpruscure" and journalItem.index >= 50 then
             WorldInstance.data.customVariables.corprusCured = true
-            madeAdjustment = true
+            return true
         elseif journalItem.quest == "a2_2_6thhouse" and journalItem.index >= 50 then
             WorldInstance.data.customVariables.corprusGained = true
-            madeAdjustment = true
+            return true 
         end
     end
 
-    return madeAdjustment
+    return false
 end
 
+---@param eventStatus boolean
+---@param pid integer Player ID
+---@param playerPacket table
 customEventHooks.registerHandler("OnPlayerJournal", function(eventStatus, pid, playerPacket)
-    if config.shareJournal == true then
-        local madeAdjustment = contentFixer.AdjustWorldCorprusVariables(playerPacket.journal)
+    if config.shareJournal == false then return end
```

Comment:
We can just check for any level of truthiness here, let's make an if not.

Reply:
To be honest, I'm just trying to keep the code as close as possible to the original as possible. Last time I checked for any level of truthiness, people died. This Lua code-base needs heavy refactoring after extraction of the server processes from lua. This would be the most opportune time to implement better practises.